meeshkan/sources/abstract.py

```diff
@@ -25,3 +26,9 @@ def shutdown(self) -> None:
         """Shutdown source.
         """
         raise NotImplementedError("")
+
+    @abc.abstractmethod
+    def initial_openapi_spec(self) -> OpenAPIObject:
```

Comment:
Hmm, I'm not sure it's the `AbstractSource`'s responsibility to define where the recordings comes from? I'd say those should be decoupled tasks, determining the source of the base schema and the source of recordings.

Reply:
It's very likely they come from different places, I think.

Reply:
What would be, in your opinion, a better way to design this?